lib/engine/game/g_18_norway/steps/nationalize_corporation.rb

```diff
@@ -12,6 +12,7 @@ def description
           def actions(entity)
             return [] if !entity.corporation? || entity != current_entity
             return [] if @game.nationalized?(entity.corporation)
+            return [] if @game.round.is_a?(G18Norway::Round::Nationalization) && @game.round.nationalization_complete
```

Comment:
This step only exists in nationalisation rounds, so the test for the round type is pointless. And there's a `@round` variable you can use instead of `@game.round`.
```suggestion
            return [] if @round.nationalization_complete
```